modules/widgets/customizer-utils.js

```diff
@@ -1,4 +1,4 @@
-/* global wp, gapi, FB, twttr */
+/* global wp, gapi, FB, twttr, PaypalExpressCheckout */
```

Comment:
🤔 what does this commenting convention mean?

Reply:
Global dependencies that need to be loaded for this script?

Reply:
It's for ESLint's `no-undef`: https://eslint.org/docs/rules/no-undef